bundle/src/main/java/com/adobe/acs/commons/errorpagehandler/cache/impl/ErrorPageCacheImpl.java

```diff
@@ -98,6 +96,7 @@ public String get(final String path,
             }
 
             cacheEntry.setData(data);
+            cacheEntry.setContentType(request.getContentType());
```

Comment:
`request.getContentType()` returns the content type of the request body. I don't think that's what you want. As far as I can tell, getting the response content type would require a new method on `ResourceDataUtil` which would return a tuple of the response body and the response content type.